src/fragments/lib/storage/flutter/upload.mdx

```diff
@@ -1,69 +1,81 @@
-To upload to S3 from a data object, specify the key and the file to be uploaded. A file can be created locally, or retrieved from the user's device using a package such as [image_picker](https://pub.dev/packages/image_picker) or [file_picker](https://pub.dev/packages/file_picker).
+## Upload File
+
+To upload to S3 from a file, specify the key and the local file to be uploaded. A file can be created locally, or retrieved from the user's device using a package such as [image_picker](https://pub.dev/packages/image_picker) or [file_picker](https://pub.dev/packages/file_picker).
+
+### Upload a local file
 
 <BlockSwitcher>
 
-<Block name="Create File">
+<Block name="Stable (Mobile)">
 
 import flutter0 from "/src/fragments/lib/storage/flutter/upload/upload-create-file.mdx";
 
 <Fragments fragments={{flutter: flutter0}} />
 
 </Block>
 
-<Block name="With Options">
+<Block name = "Developer Preview (Mobile & Desktop)">
+
+```dart
+import 'dart:io' as io;
+import 'package:amplify_core/src/io/aws_file_platform_io.dart';
```

Comment:
We should be exporting this if it's the only way to do it. Having them import a `src/` file is gonna trigger linter warnings for them and generally means we've done something wrong.